src/nvim/normal.c

```diff
@@ -341,6 +341,7 @@ static const struct nv_cmd {
   {K_F8,      farsi_fkey,     0,                      0},
   {K_F9,      farsi_fkey,     0,                      0},
   {K_EVENT,   nv_event,       NV_KEEPREG,             0},
+  {K_COMMAND,   nv_command,       NV_KEEPREG,             0},
```

Comment:
This also looks misaligned. You will also have to add spaces before `}` and after `{` everywhere, better in the separate commit. Otherwise linter will fail.